awx/settings/defaults.py

```diff
@@ -1041,26 +1041,26 @@ def IS_TESTING(argv=None):
             'class': 'django.utils.log.AdminEmailHandler',
         },
         'tower_warnings': {
-            'level': 'WARNING',
-            'class':'logging.handlers.RotatingFileHandler',
+            # don't define a level here, it's set by settings.LOG_AGGREGATOR_LEVEL
```

Comment:
Awesome comment! It would also apply to the "external_logger" handler, so it would be good to add it there as well. Eventually.